.github/workflows/pr.yaml

```diff
@@ -46,14 +48,30 @@ jobs:
     uses: rapidsai/shared-workflows/.github/workflows/conda-python-build.yaml@branch-24.12
     with:
       build_type: pull-request
-      # This selects "ARCH=amd64 + the latest supported Python + CUDA".
-      matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
+  conda-python-tests:
+    needs: [conda-python-build, changed-files]
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-python-tests.yaml@branch-24.12
+    if: fromJSON(needs.changed-files.outputs.changed_file_groups).test_python
+    with:
+      build_type: pull-request
+      run_codecov: false
```

Comment:
Saw codecov upload failing like this:

> error - 2024-11-08 16:32:44,232 -- Commit creating failed: {"message":"Repository not found"}

([build link](https://github.com/rapidsai/nx-cugraph/actions/runs/11745428665/job/32723131660))

Maybe this repo needs to be set up in the codecov UI. But since @eriknw mentioned that codecov isn't really event being used in development of this project right now, I'm proposing just skipping it.